### ISSUE TITLE
Robot closes down when Grasshopper/Rhino closes/adapter is released

### ISSUE THREAD

**Issue by rwemay** [BHoM/Robot_Toolkit#72]
Robot should stay open after any BHoM adapter session.

**Comment by IsakNaslundBh** [comment]
@rwemay I think this only happends if Robot is started with the adapter, right?

if you manually start Robot, I think it should stay open after the session?

**Comment by rwemay** [comment]
Yes. I would start Robot before initiating from within Grasshopper/outside of Robot to avoid this. We tried previously to fix this but couldn't immediately find a solution.
